client/src/components/Channel.js

```diff
@@ -168,9 +168,9 @@ class Channel extends React.Component {
       ChannelActions.sendMessage(this.state.channelName, text);
   }
 
-  sendFile(filePath: string, buffer) {
+  sendFile(filePath: string, buffer, mimeType = 'application/octet-binary') {
```

Comment:
Do you want to add that or should i? I'm not sure if we should have an explicit `mimeType` argument or just a more generic object that we can use for more meta data like encoding and whatever else might be useful in the future, i.e. `{ name, hash, size, meta = { mimeType, encoding } }` vs. `{ name, hash, size, mimeType }`